tests/api/cars/createCar.spec.js

```diff
@@ -0,0 +1,46 @@
+import {expect, test} from "../../../src/fixtures/myFixtures";
+import {CAR_BRANDS} from "../../../src/data/carBrands";
+import {CAR_MODELS} from "../../../src/data/carModels";
+import {faker} from "@faker-js/faker";
+
+test.describe("Car with controller", ()=>{
+
+    test.afterEach(async ({request})=>{
+        const carsList = await request.get('/api/cars')
+        const {data: cars} = await carsList.json()
+
+        for (const car of cars) {
+            const res = await request.delete(`/api/cars/${car.id}`)
+            await expect(res).toBeOK()
+        }
+    })
```

Comment:
This way you delete all cars what can affect other tests

delete only car I have created